frontend/src/theme.ts

```diff
@@ -29,6 +29,19 @@ export const theme = responsiveFontSizes(
         fontSize: '1.5rem',
         fontWeight: 600,
       },
+      fontFamily: [
+        'Poppins',
+        '-apple-system',
+        'BlinkMacSystemFont',
+        '"Segoe UI"',
+        'Roboto',
+        '"Helvetica Neue"',
+        'Arial',
+        'sans-serif',
+        '"Apple Color Emoji"',
+        '"Segoe UI Emoji"',
+        '"Segoe UI Symbol"',
+      ].join(','),
```

Comment:
Nice for consistency.  
Note that using a custom font as the default font is sometimes discouraged, as it may be rendered slightly differently by different systems and be uncomfortable for the eyes, especially for longer text content. We'll see how the change is appreciated on different devices.

Reply:
How is it different if the font Poppins is the default rather than setting it manually on every components?

Reply:
It's not different, except after this change the font is now applied in every form, Typography blocks, etc. (where a system font was used until now).